app/modules/asset_groups/models.py

```diff
@@ -186,10 +186,14 @@ def commit(self):
 
         # Create sighting in EDM
         try:
+            # Don't send the encounter guids. EDM doesn't use them but they confuse anyone reading logs
+            sent_data = self.config
+            for enc in sent_data['encounters']:
+                enc.pop('guid')
             result_data = current_app.edm.request_passthrough_result(
                 'sighting.data',
                 'post',
-                {'data': self.config, 'headers': {'Content-Type': 'application/json'}},
+                {'data': sent_data, 'headers': {'Content-Type': 'application/json'}},
```

Comment:
Huh. Did we need them before for some reason or were they always extraneous?

Reply:
They were always extraneous. These were the temporary encouner IDs that could be used to update the encounter metadata in the AGS before the real one was created.